motion/los_guidance/scripts/los_guidance_backstepping.py

```diff
@@ -491,7 +491,8 @@ def config_callback(self, config, level):
 
 		delta = config['delta']
 		
-		rospy.loginfo("""los_guidance reconfigure request: \n delta: {0} -> {1}""".format(self.los.delta, delta))
+		rospy.loginfo("los_guidance reconfigure:")
+		rospy.loginfo("\t delta: {:.4f} -> {:.4f}".format(self.los.delta, delta))
```

Comment:
can you explain this code a little bit?

Reply:
Just text formatting :smile: The empty string before the : means "take the next argument in the format() call, and the .4f means "print with a precision of 4 decimal points.